### PR TITLE
Login refactor

### PR DESCRIPTION
**Pull request by sperBruno**
Hi @carledriss @HenrrySalinasJala @MijhailVillarroel @DanielRGonzalesV @rosariogarcia. Could you guys please make this pull request a code review?

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -36,7 +36,7 @@ repositories {
 dependencies {
     // The production code uses the SLF4J logging API at compile time
     compile 'org.slf4j:slf4j-api:1.7.12'
-	compile 'org.seleniumhq.selenium:selenium-java:2.46.0'
+	compile 'org.seleniumhq.selenium:selenium-java:2.53.0'
```
Comment (carledriss) [inline build.gradle]: in fact. the last version is 2.53.1
Change it.

File: gradle.properties [hunk 2]
```diff
@@ -2,11 +2,11 @@ rootProject.name = 'SFDC-Team1'
...
 suites = src/test/resources/TestNGSuite.xml
-primaryUserName =
-primaryUserPassword =
-primaryUsePasswordToken =
+primaryUserName =henrry.salinas@fundacion-jala.org
+primaryUserPassword =P4ssw0rdd403
+primaryUsePasswordToken =k8X6slKc1N7BX8duRmvMLUsH
 urlApi = https://login.salesforce.com/services/Soap/u/34.0
-displayName = veronica prado
+displayName = henrry salas
```
Comment (carledriss) [inline gradle.properties]: Remove the accounts and passwords

File: src/test/resources/TestNGSuite.xml [hunk 3]
```diff
@@ -5,36 +5,7 @@
     </listeners>
     <test name="Full">
         <classes>
-             <class name="com.salesforce.dev.TestNGSuite"/>
```
Comment (carledriss) [inline src/test/resources/TestNGSuite.xml:8]: Don't remove all the tests

**Comment by carledriss** [comment]
@sperBruno 
After fixing the observations
Looks good to me!

**Comment by sperBruno** [comment]
@carledriss I have already corrected the observation you made to this pull request.
thanks.

**Pull request merged**